### DISCUSSION TITLE
Can i create 2 stacks on the same domain?

### DISCUSSION THREAD

**Discussion by saemkoch** [mozilla/hubs#3367] [Hubs Cloud Questions] [answered]
I am trying to create another cloudformation stack on the same domain and it keeps doing the rollback process. Can I create 2 stacks on the same domain? I already had the enterprise on this domain but I would like to create another one as a personal stack.
Much appreciate it if anyone can help to answer this.
Saem

**Comment by robinkwilson** [comment, marked as answer]
I'm afraid you can't the way it is right now. You could use the same domain for the main domain but have two different subdomains. But the internal domain + link domain have to be different.

Example:
hub1.**domain.com** - same base domain, different subdomain
internaldomain1.com - needs to be different
linkdomain1.link - needs to be different

hub2.**domain.com** - same base domain, different subdomain
internaldomain2.com - needs to be different
linkdomain2.link - needs to be different

**Comment by saemkoch** [comment]
Hi Robin
Thank for the reply.
If I wanna change from m Enterprise to Personal formation on the same
domain I will have to delete the enterprise formation and create a Personal
formation right?.
Regards
Saem

**Reply by robinkwilson** [reply]
[Backup and Restore](https://hubs.mozilla.com/docs/hubs-cloud-aws-backup-and-restore.html) if you want to save your data + switch to a Personal stack.
OR 
**In your current Enterprise template make it equivalent to a Personal setup** 
Update the stack: **select 1 app server** and **0 streaming servers**  + **enable database pausing** = Personal stack using the Enterprise template _-- Personal + Enterprise are the same except defaults and costs + Enterprise allows you to use multiple server setups (Personal is always 1 server)_
OR 
Delete the stack completely and start a fresh new Personal one with no data.

**Comment by saemkoch** [comment]
Thank you Robin for the advice.
Best regards
SAem

On Thu, Dec 3, 2020 at 12:11 AM Robin K Wilson <notifications@github.com>
wrote:

> Backup and Restore
> <https://hubs.mozilla.com/docs/hubs-cloud-aws-backup-and-restore.html> if
> you want to save your data + switch to a Personal stack.
> OR
> In your current Enterprise template make it equivalent to a Personal
> setup: Update the stack and *select 1 app server* and *0 streaming
> servers* + *enable database pausing* = Personal stack using the
> Enterprise template *-- Personal + Enterprise are the same except
> defaults and costs + Enterprise allows you to use multiple server setups
> (Personal is always 1 server)*
> OR
> Delete the stack completely and start a fresh new Personal one with no
> data.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/hubs/discussions/3367#discussioncomment-145107>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUKBAZ3KLMIDLPHGOTZCULSSZYLDANCNFSM4TZSNLFQ>
> .
>


-- 
Best Regards
Saem Koch Mr.
General  Manager

Cityneon Displays&Constructions (Thailand)Co.,Ltd.
436/87-94 TC Tower 20 Mithuna Rd. Huaykwang Bangkok 10310
Tel:662 690 2682
Fax:662 690 2685
Mobile:669 788 2909
Email: saemkoch@cityneonthailand.com
www.cityneonthai.com
